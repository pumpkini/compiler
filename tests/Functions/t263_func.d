int h() {
    Print("h");
    return 1;
}

int g() {
    Print("g");
    h();
    return 1;
}

int f(){
    Print("f");
    g();
    h();
    return 3;
}

int main()  {
    f();
    g();
    h();
}