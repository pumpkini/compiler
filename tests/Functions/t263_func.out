f
g
h
h
g
h
h
